README.md

```diff
@@ -31,7 +31,7 @@ THIS SOFTWARE.
 ## Distribution and Reporting Problems
 
 Changes, mostly bug fixes and occasional enhancements, are listed
-in `FIXES`.  If you distribute this code further, please please please
+in `FIXES`.  If you distribute this code further, please, please, please
```

Comment:
Happy to drop, but this happens to both be grammatically/stylistically correct and make my tooling happier.